Components/Typography/typography.css

```diff
@@ -0,0 +1,44 @@
+.typography-box {
+  border: 1px solid var(--gray);
+  margin: auto;
+  padding: 1.5rem;
+  border-radius: 6px;
+  margin-bottom: 1.5rem;
+}
+.type-1,
+.type-2,
+.type-3,
+.type-4,
+.type-5,
+.type-6 {
```

Comment:
better naming can be done, `type` doesn't tell it's for font or image or what. If font then what type of font heading, regular, small